packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { ReactElement, useMemo } from 'react';
+import classNames from 'classnames';
+import Link from 'next/link';
+import { Notification } from '../../graphql/notifications';
+import { useDomPurify } from '../../hooks/useDomPurify';
+import NotificationItemIcon from './NotificationIcon';
+import NotificationItemAttachment from './NotificationItemAttachment';
+import NotificationItemAvatar from './NotificationItemAvatar';
+
+export interface NotificationItemProps
+  extends Pick<
+    Notification,
+    'type' | 'icon' | 'title' | 'description' | 'avatars' | 'attachments'
+  > {
+  isUnread?: boolean;
+  targetUrl: string;
+  onClick?: (e: React.MouseEvent<HTMLAnchorElement>) => void;
+}
+
+function NotificationItem({
+  icon,
+  title,
+  isUnread,
+  description,
+  avatars,
+  attachments,
+  targetUrl,
+  onClick,
+}: NotificationItemProps): ReactElement {
+  const purify = useDomPurify();
+  const { title: memoizedTitle, description: memoizedDescription } =
+    useMemo(() => {
+      if (!purify?.sanitize) {
+        return { title: '', description: '' };
+      }
+
+      return {
+        title: purify.sanitize(title),
+        description: purify.sanitize(description),
+      };
+    }, [purify, title, description]);
+
+  if (!purify) {
+    return null;
+  }
+
+  const avatarComponents =
+    avatars
+      ?.map?.((avatar) => (
+        <NotificationItemAvatar key={avatar.referenceId} {...avatar} />
+      ))
+      .filter((avatar) => avatar) ?? [];
+  const hasAvatar = avatarComponents.length > 0;
+
+  return (
+    <Link href={targetUrl} passHref>
+      <a
+        href="#"
```

Comment:
What should this be?

Reply:
This is overridden by the `passHref` from the `Link` component.

Reply:
I think in that case we can either omit this href, right?
Or simply pass the same href just for linting purpose.

Reply:
Hmm. Why I didn't think of just passing the same `targetUrl` as you mentioned in the first place 🤣